### ISSUE TITLE
[API] Revise draft API for consistency

### ISSUE THREAD

**Issue by VWoeltjen** [nasa/openmct#1110]
Review and revise draft API from #999 (once that is complete, in terms of checklists at least) for consistency. In particular, want to look at:

* Naming conventions (e.g. casing, plurals, verbosity, etc)
* Patterns, particularly for registration (are we doing same/similar things in different ways? Is there a good reason?)
* Implementation details, such as how properties/methods are attached to the `mct` object (lower-priority, since this could also be deferred in principle)

**Comment by VWoeltjen** [comment]
Looking at this in concert with #1111; that JSDoc will be useful for assessing both consistency and completeness of the prototype API

**Comment by larkin** [comment]
Closed as handled in #1124.
